ibis/expr/types/generic.py

```diff
@@ -17,6 +17,7 @@
     import pyarrow as pa
 
     import ibis.expr.types as ir
+    import ibis.expr.builders as bl
```

Comment:
Is this additional import here necessary?

Reply:
Oops, I amended the commit and the point of this import disappeared. In new version it is needed.